<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>高颢嘉</author>
 <class>Sign_in_window</class>
 <widget class="QMainWindow" name="Sign_in_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">::Sign_in_window
{
border-image:url(:/sign_in/OIP-C.jpg)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>考试系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_Inputer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_Inputer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="Teacher_QRadio">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>老师</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Student_QRadio">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_in_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_up_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="background_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background_image</zorder>
   <zorder>label</zorder>
   <zorder>account_Inputer</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>Password_Inputer</zorder>
   <zorder>Teacher_QRadio</zorder>
   <zorder>Student_QRadio</zorder>
   <zorder>sign_in_button</zorder>
   <zorder>sign_up_button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
